doc/devel/release_guide.rst

```diff
@@ -112,7 +112,7 @@ For the "what's new",
 Similarly for the "API changes",
 
  1. copy the current api changes to a file is :file:`doc/api/prev_api_changes`
- 2. merge all of the files in :file:`doc/api/next_api_changes/` into
+ 2. merge all of the files in :file:`doc/api/recent_api_changes_3.3/` into
```

Comment:
This needs to updated to include instructions on how to create the folder an files for the next version.